deployment/src/main/java/org/wso2/carbon/testgrid/deployment/DeployerConstants.java

```diff
@@ -0,0 +1,14 @@
+package org.wso2.carbon.testgrid.deployment;
+
+public class DeployerConstants {
+    //Constants needed to run deploy.sh
+    public static final String USERNAME = System.getenv("OS_USERNAME");
+    public static final String PASSWORD = System.getenv("OS_PASSWORD");
+    public static final String DOCKER_URL = "dockerhub.private.wso2.com";
```

Comment:
It would be better to add "WSO2" prefix in Docker-related constants.

Reply:
Fixed in 7e22a24